Sources/LanguageServerProtocol/CodeAction.swift

```diff
@@ -47,6 +48,30 @@ public struct CodeActionRequest: TextDocumentRequest, Hashable {
     self.context = context
     self.textDocument = textDocument
   }
+
+  public func injectMetadata(atResponse response: CodeActionRequestResponse?) -> CodeActionRequestResponse? {
```

Comment:
This implementation is not really a general feature of CodeAction, so it should probably move into an extension the SourceKit module.